services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -101,8 +340,15 @@
 	<bean id="replicationProcessor" class="edu.unc.lib.cdr.ReplicationProcessor">
 		<constructor-arg ref="repository"/>
 		<constructor-arg value="${cdr.replication.locations}" />
+	</bean>
+	
+	<bean id="solrIngestProcessor" class="edu.unc.lib.cdr.SolrIngestProcessor">
+		<constructor-arg ref="repository"/>
 		<constructor-arg value="${error.maxRedeliveries}" />
 		<constructor-arg value="${error.retryDelay}" />
+		<constructor-arg ref="factory" />
```

Comment:
I don't think "factory" exists?  You probably need to create a DocumentIndexingPackageFactory.  Also, give it a more informative name